src/Razor/src/Microsoft.AspNetCore.Razor.LanguageServer/Completion/VSCompletionListCapability.cs

```diff
@@ -6,5 +6,7 @@ namespace Microsoft.AspNetCore.Razor.LanguageServer.Completion
     internal class VSCompletionListCapability
     {
         public bool CommitCharacters { get; set; }
+
+        public bool Data { get; set; }
```

Comment:
Same comment as above, more details about what "Data" means here, or a more descriptive name.

Reply:
Same answer as before; however, in this scenario it's a `bool` because it's a capability. It represents whether or not the VS LSP client can support having a `Data` property on the completion list.